services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -724,7 +725,15 @@ protected void initEth1DataCache() {
 
   protected void initAttestationTopicSubscriber() {
     LOG.debug("BeaconChainController.initAttestationTopicSubscriber");
-    this.attestationTopicSubscriber = new AttestationTopicSubscriber(spec, p2pNetwork);
+    final SettableLabelledGauge subnetSubscriptionsGauge =
+        SettableLabelledGauge.create(
+            metricsSystem,
+            TekuMetricCategory.NETWORK,
+            "subnet_subscriptions",
+            "Tracks attestations subnet subscriptions",
+            "attestations");
```

Comment:
I'm not very familiar with the metrics, but I see that we usually put something like generalization keyword here, `type` or whatever. So my guess is that `type` or `subnet` would be better for the label, but I maybe wrong

Reply:
Yeah you're right. Changed the label to "type"